packages/block-library/src/navigation/index.php

```diff
@@ -271,12 +267,18 @@ function register_block_core_navigation() {
 				'customTextColor'       => array(
 					'type' => 'string',
 				),
+				'valueTextColor'        => array(
```

Comment:
Which "value" is that? Could we find a more descriptive name here?

Reply:
This value (already removed) came from the use-colors hook. I had modified introduced the ability to set up this attribute from there, too.
The reason was to be able to pick up the color value of the text, independently if it was defined using a custom color or a palette color, to apply to the border-color property.
I've removed all this stuff and replaced using the CSS currentColor property.

Reply:
I've restored these changes. Going to explain why in the use-colors hook change.

Reply:
Basically this attribute will store the color value beyond how it was defined, either picking up from the colors palette or using the custom color panel.